srcpkgs/cosmic-applets/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cosmic-applets'
+pkgname=cosmic-applets
+version=1.0.0.alpha.1
+revision=1
+_tag=epoch-${version/.alpha/-alpha}
+build_style=cargo
+hostmakedepends="pkg-config just"
+makedepends="libxkbcommon-devel eudev-libudev-devel dbus-devel pulseaudio-devel libinput-devel"
+short_desc="TODO: cosmic-applets short_desc"
+maintainer="Daniel Martinez <danielmartinez@cock.li>"
+license="GPL-3.0-only"
+homepage="https://github.com/pop-os/cosmic-applets"
+distfiles="https://github.com/pop-os/cosmic-applets/archive/refs/tags/${_tag}.tar.gz"
+checksum=be60cba82b8da4f7d7cde4a2a5426d97f41423c11bd5fb5d66f1194ddf7274d4
+
+if [ "$XBPS_WORDSIZE" = "32" ]; then
+	broken="rustc-LLVM ERROR: out of memory"
+fi
```

Comment:
This only occurs when building in an i686 masterdir: `./xbps-src -A i686 pkg cosmic-applets`

Does not occur when builing in an x86_64-glibc masterdir, and cross compiling to i686: `./xbps-src -A x86_64 -a i686 pkg cosmic-applets`